playbooks/trust/del-trust.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Playbook to delete trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+    - name: delete the trust with windows.local
```

Comment:
States are ensured by Ansible. Use "ensure trust is absent".

Reply:
done